### PR TITLE
Add visual regression coverage for agSelect styles

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-ag-select-hover-style-e2e](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-ag-select-hover-style-e2e)

File: e2e/editable-grid.e2e-spec.ts [hunk 1]
```diff
@@ -270,3 +270,93 @@ describe('Editable grid', () => {
...
+  // selectors
+  const selectCell = '.ag-body-viewport [aria-colindex="2"]';
+  const selectCellTrigger = '.ag-body-viewport [aria-colindex="2"] .ag-picker-field-display';
+  const selectList = '.ag-select-list';
+  const editButton = '#edit-btn';
+
+  let currentTheme: string;
+  let currentThemeMode: string;
+
+  async function selectTheme(theme: string, mode: string): Promise<void> {
```
Comment (Blackbaud-TrevorBurch) [inline e2e/editable-grid.e2e-spec.ts:285]: Minor - but could we just move the first describe block’s `selectTheme` and `getScreenshotName` above the first `describe` block? Then we wouldn’t need this duplicate code.

File: src/app/public/styles/ag-grid-styles.scss [hunk 2]
```diff
@@ -421,6 +421,26 @@ ag-grid-angular.sky-ag-grid-editable {
       }
     }
   }
+
+  .ag-popup {
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/styles/ag-grid-styles.scss:425]: Why are we adding new styles in this PR?

**Review by Blackbaud-TrevorBurch — COMMENTED** [review]
Just a couple questions.

File: src/app/public/styles/ag-grid-styles.scss (hunk 2 above)
Comment (johnhwhite) [reply src/app/public/styles/ag-grid-styles.scss:425]: There were issues with the dark theme that I noticed when reviewing the e2e results.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-ag-select-hover-style-e2e](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-ag-select-hover-style-e2e)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**